### PR TITLE
docs: 2017-03-08 weekly meeting

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/ctc @bmeck 

a bit delayed this week cause wanted to make sure I captured the module discussion properly. thanks!

### CODE REVIEW

File: meetings/2017-03-08.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+### governance: add Franziska Hinkelmann to the CTC [node#11488](https://github.com/nodejs/node/pull/11488)
+
+**VOTE (in progress)**
+* In favor (14): @addaleax, @bnoordhuis, @ChALkeR, @cjihrig, @evanlucas,
+  @Fishrock123, @jasnell, @mhdawson, @misterdjules, @MylesBorins, @ofrobots,
+  @targos, @thefourtheye, @Trott
+* Opposed (0):
+* Abstain (0):
+* Not yet voted (5): @indutny, @mscdex, @rvagg, @shigeki, @trevnorris
```
Comment (Trott) [inline meetings/2017-03-08.md]: Since @rvagg opened the pull request to add Franziska, it's probably safe to move him to "In favor".

**Review by Trott — APPROVED** [review]
LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by joshgav** [comment]
landed in 20d06a13b1038fa91df63fdae36bcc6533b9b879. Thanks!

**Pull request closed without merging**